model.go

```diff
@@ -8,14 +6,64 @@ import (
 	// "errors"
 )
 
-
 // tom: add backticks to json
 type product struct {
 	ID    int     `json:"id"`
 	Name  string  `json:"name"`
 	Price float64 `json:"price"`
 }
 
+type storeProduct struct {
+	Name        string  `json:"name"`
+	ProductID   int     `json:"product_id"`
+	Price       float64 `json:"price"`
+	IsAvailable bool    `json:"is_available"`
+}
+
+type store struct {
+	ID          int  `json:"id"`
+	ProductID   int  `json:"product_id"`
+	IsAvailable bool `json:"is_available"`
+}
+
+func getStoreProducts(db *sql.DB, id int) ([]storeProduct, error) {
+	productsInStore := []storeProduct{}
+
+	query := `SELECT p.NAME, s.product_id, p.price, s.is_available 
+			FROM products AS p 
+			JOIN store AS s ON p.ID = s.product_id
+        	WHERE s.id = $1
+        	`
+	rows, err := db.Query(query, id)
+	if err != nil {
+		return productsInStore, err
+	}
+
+	defer rows.Close()
+
+	for rows.Next() {
+		var sp storeProduct
+		if err := rows.Scan(&sp.Name, &sp.ProductID, &sp.Price, &sp.IsAvailable); err != nil {
+			return productsInStore, err
+		}
+		productsInStore = append(productsInStore, sp)
+	}
+
+	return productsInStore, nil
+}
+
+func (s *store) createStore(db *sql.DB) error {
```

Comment:
Since products are being added to store, can you consider changing the method name to **addProductsToStore**.

Reply:
Renamed the function to addProductToStore - Ref: [Commit 228b0e](https://github.com/nitinks-dnz/go-mux/commit/228b0eb1d2fb8c1bdbebaaedc983c36e7d04ca0d)